packages/botonic-core/src/hubtype-service.js

```diff
@@ -1,9 +1,18 @@
 import axios from 'axios'
 import Pusher from 'pusher-js'
 
-const PUSHER_KEY = process.env.WEBCHAT_PUSHER_KEY || '434ca667c8e6cb3f641c'
-const HUBTYPE_API_URL = process.env.HUBTYPE_API_URL || 'https://api.hubtype.com'
-
+const PUSHER_KEY =
+  (typeof WEBCHAT_PUSHER_KEY !== 'undefined'
```

Comment:
where is WEBCHAT_PUSHER_KEY defined?

Reply:
`WEBCHAT_PUSHER_KEY` comes from:
```
new webpack.DefinePlugin({
        IS_BROWSER: true,
        HUBTYPE_API_URL: JSON.stringify(process.env.HUBTYPE_API_URL),
        WEBCHAT_PUSHER_KEY: JSON.stringify(process.env.WEBCHAT_PUSHER_KEY),
      }),
```
when using DefinePlugin, the variable is available at compile time to do the build

Reply:
preserved same symbol, using it inside an anonymous function to handle the scope